mlflow/java/client/pom.xml

```diff
@@ -41,6 +41,14 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
```

Comment:
This dependency seems to only be used in a test, so maybe we can mark it with `<scope>test</scope>`.

Reply:
Nice!